frontend/integration-tests/views/crud.view.ts

```diff
@@ -12,6 +12,11 @@ export const createYAMLLink = $('#yaml-link');
 export const saveChangesBtn = $('#save-changes');
 export const reloadBtn = $('#reload-object');
 export const cancelBtn = $('#cancel');
+export const addStorageItem = element(by.cssContainingText('.dropdown li a', 'Add Storage'));
+export const createNewClaim = $('[value=new]');
+export const inputPVCName = $('#pvc-name');
+export const inputPVCSize = $('[name=requestSizeValue]');
+export const inputMountPath = $('#mount-path');
```

Comment:
These should go in a separate storage.view.ts file

Reply:
Good idea. Moved storage related code to storage.view.ts.